### PR TITLE
Add Event show action to the API

### PR DESCRIPTION
**Pull request by prognostikos**
Implements https://github.com/getlago/lago/issues/48
Closes https://github.com/getlago/lago/issues/48

### CODE REVIEW

**Comment by prognostikos** [comment]
The specs will probably fail due to the missing setup for active record encryption.

**Comment by vincent-pochet** [comment]
Thanks @prognostikos for this PR!

> The specs will probably fail due to the missing setup for active record encryption.

Indeed, the spec suite uses secrets from repository and Lago organization. For security reasons these secrets are not exposed to external pull request.
Since these secrets are not really sensitive in test env, we should be able to find a way to run specs without secrests

**Review by vincent-pochet — COMMENTED** [review]
One small comment, otherwise looks good! :100:

**Review by vincent-pochet — APPROVED** [review]

File: app/controllers/api/v1/events_controller.rb [hunk 1]
```diff
@@ -17,6 +17,22 @@ def create
         head(:ok)
       end
 
+      def show
+        event = Event.find_by(
+          organization: current_organization,
+          transaction_id: params[:id]
+        )
+
+        return head(:not_found) unless event
```
[the remark this answers is not in the document]
Comment (prognostikos) [reply app/controllers/api/v1/events_controller.rb]: @vincent-pochet good idea to use the same format. I've added the `not_found_error`.

**Comment by vincent-pochet** [comment]
Thank you very much @prognostikos, it will be part of the next release

**Pull request merged**